simp-beaker-helpers.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'beaker', '~> 3.14'
   s.add_runtime_dependency 'beaker-puppet', '~> 0.8.0'
   s.add_runtime_dependency 'beaker-puppet_install_helper', '~> 0.6'
+  s.add_runtime_dependency 'highline', '~> 1.6'
```

Comment:
When I built the gem and installed it in rvm 2.1.9 and then tried to run  rubygem-simp-cli spec tests, the test hung, indicating IO problems.

Reply:
battling multiple rubygem-simp-cli spec test problems which may be entirely unrelated to this.